spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "./app/models/user"
+
+describe User do
+  context "when all attributes are correct" do
+    it "is created" do
+      user = User.create(name: "John Doe",
```

Comment:
Creating a user in the database is expensive.
Instead I would:
1. build a user and check if it is valid?
2. use shoulda-matchers

I would go for option 2